### PR TITLE
Fix some typo, grammar and word choice issues

### PR DESCRIPTION
**Pull request by redagavin**
I just fixed some small mistakes which do not influence the content.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -30,14 +30,14 @@ See:
...
-* Lets you choose the size of the QR code and customize things.
-* Is completly internationalized (i18n).
-* Let's you use a shortcut (Ctrl+Shift+F10) for generating the QR code.
-* Generate QR codes from selected text on the website.
-* Complete Unicode/UTF-8/Emoji support.
+* You can choose the size of the QR code and customize things.
+* Is completely internationalized (i18n).
+* You can use a shortcut (Ctrl+Shift+F10) for generating the QR code.
+* Generates QR codes from selected text on the website.
+* Completes Unicode/UTF-8/Emoji support.
```
Comment (rugk) [inline README.md]: No, here, that's not supposed to be a verb. 😄

Comment (redagavin) [reply README.md]: which one? Line 40? the word "complete"?

Comment (rugk) [reply README.md]: Yes, it should say it has "full Unicode support", i.e. you can copy Emojis or so in there, not that it somehow "completes" the support.

Comment (redagavin) [reply README.md]: I see.

**Comment by redagavin** [comment]
I have made the change you requested and proofread the files in asset/en/

**Comment by redagavin** [comment]
@rugk

**Comment by rugk** [comment]
Yeah, sorry, I lost track of this. I'll have a look soon.

**Pull request merged**